### PR TITLE
Fix lint pre-commit hook and scripts

### PR DESCRIPTION
**Pull request by CapnMigraine**
Run lint-staged pre-commit hook
Fix lint and lint-fix scripts

Ticket: BG-18037

### CODE REVIEW

File: src/coin/trx/iface.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ export interface TransactionReceipt {
 export interface RawData {
   expiration: number;
   timestamp: number;
-  contractType: ContractType;
+  contractType?: ContractType;
```
Comment (CapnMigraine) [inline src/coin/trx/iface.ts]: @argjv - not 100% sure on this one. `UnsignedBuildTransaction` in `test/resources/trx:13` doesn't have a `contractType`

File: src/coin/xtz/transaction.ts [hunk 2]
```diff
@@ -80,7 +80,7 @@ export class Transaction extends BaseTransaction {
           break;
         default:
           break;
-          // throw new ParseTransactionError('Unsupported contract type');
+        // throw new ParseTransactionError('Unsupported contract type');
```
Comment (argjv) [inline src/coin/xtz/transaction.ts]: My bad. Can you remove this line?

File: src/coin/trx/iface.ts (hunk 1 above)
Comment (argjv) [reply src/coin/trx/iface.ts]: I think it's fine. It is missing sometimes

File: src/coin/xtz/transaction.ts (hunk 2 above)
Comment (CapnMigraine) [reply src/coin/xtz/transaction.ts]: done

**Pull request merged**